README.md

```diff
@@ -1 +1,68 @@
-# React-Calculator
\ No newline at end of file
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
**Kindly update your `README` file and remove unnecessary content**.
Checkout out the below link for help, free free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/